src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala

```diff
@@ -89,4 +87,17 @@ object RoutesRepoHelper extends DaoHelper {
         |      }]
         |  }]
         |}""".stripMargin
+
+  private def insertTestPath(ownedByTeam: String, createdBy: String, createdAt: LocalDateTime): Long = {
```

Comment:
you could split the parameters on more lines :) maybe this should be the default...